tox.ini

```diff
@@ -6,6 +6,8 @@ ignore=
     W503,
     ; line break after binary operator
     W504
+    ; `.next()` is not a thing on Python 3 (it is for TaskActionTimer)
+    B305
```

Comment:
This rule checks for `.next()` calls which are a common 2to3 error, however, it checks somewhat naively, our usage is valid, don't think the rule is worth it.